### PR TITLE
Add HearthstoneJSON data

### PR DESCRIPTION
**Pull request by FuriKuri**
Hi,

I've added a JSON dataset for all hearthstone fans ;-)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ A curated list of awesome JSON datasets that don't require authentication.
 * [Gaming](#gaming)
 * [GitHub API](#github-api)
 * [Government](#government)
+* [Hearthstone] (#hearthstone)
```
Comment (jdorfman) [inline README.md]: @FuriKuri This could just go under "Gaming"

File: README.md [hunk 2]
```diff
@@ -308,6 +309,13 @@ You can also change the country e.g. `sold_in_uk=1`, `sold_in_de=1`, etc.
   * [Noosa Wedding Locations](http://data.gov.au/geoserver/noosa-wedding-locations/wfs?request=GetFeature&typeName=17a54ec7_914b_4c60_b194_b0d190803001&outputFormat=json)
   * [NSW State Electoral Boundaries - PSMA Administrative Boundaries](http://data.gov.au/geoserver/nsw-state-electoral-boundaries-psma-administrative-boundaries/wfs?request=GetFeature&typeName=8047ddd1_7193_4667_aef9_b75bc3076075&outputFormat=json)
 
+## Hearthstone
```
Comment (jdorfman) [inline README.md]: Move under gaming and we are golden =)

File: README.md (hunk 1 above)
Comment (jdorfman) [inline README.md]: Correction: Remove this Category and move the datasets under gaming. Thanks!

**Comment by FuriKuri** [comment]
My mistake. I misunderstood the struture.
I've moved it to gaming.

**Comment by jdorfman** [comment]
Hey @FuriKuri please make an individual pull request for each suggestion. Closing this, please resubmit. =)

**Pull request closed without merging**